keyboards/jj4x4/README.md

```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
 
     make jj4x4:default:flash
 
-**Reset Key**: Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard.
+**Reset Key**:
+
+Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

Comment:
```suggestion
Hold down the key located at *K12*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

My JJ4x4 arrived, it is indeed this key, but it is labelled K12 in the macro define.